src/pages/Bridge/BridgeModal/BridgeModal.tsx

```diff
@@ -78,6 +78,11 @@ export const BridgeModal = ({
       setIsWarning(false)
       setBridgeName('Arbitrum One Bridge')
     }
+
+    if (activeBridge === 'connext') {
+      setIsWarning(false)
+      setBridgeName('Connext Network')
+    }
```

Comment:
should we have some enum for all avaliable bridges instead of just hardcoind it like this?

Reply:
Basically `bridgeId` is typed as `BridgeList` so we don't hardcode it.
```ts
// EcoBridge.types.ts
export type ConnextList = 'connext'
export type BridgeList = ArbitrumList | SocketList | ConnextList


// EcoBridge.utils.ts
export abstract class EcoBridgeChildBase {
  public readonly bridgeId: BridgeList
}

// EcoBridge.config.ts
export const ecoBridgeConfig: EcoBridgeChildBase[] = [
  new Connext({
    bridgeId: 'connext',
    displayName: 'Connext',
    supportedChains: [
      { from: ChainId.MAINNET, to: ChainId.ARBITRUM_ONE },
      { from: ChainId.MAINNET, to: ChainId.XDAI },
      { from: ChainId.MAINNET, to: ChainId.POLYGON },
      { from: ChainId.XDAI, to: ChainId.ARBITRUM_ONE },
      { from: ChainId.XDAI, to: ChainId.POLYGON },
      { from: ChainId.POLYGON, to: ChainId.ARBITRUM_ONE },
    ],
  }),
]


```